src/main/java/org/opentripplanner/graph_builder/module/osm/OpenStreetMapModule.java

```diff
@@ -762,6 +777,22 @@ private VehicleParking createVehicleParkingObjectFromOsmEntity(
         .build();
     }
 
+    private OHCalendar parseOpeningHours(OSMWithTags entity) {
+      final var openingHoursTag = entity.getTag("opening_hours");
+      if (openingHoursTag != null) {
+        try {
+          return osmOpeningHoursParser.parseOpeningHours(openingHoursTag);
+        } catch (OpeningHoursParseException e) {
+          issueStore.add(
+            "OSMOpeningHoursUnparsed",
+            "OSM object with id '%s' has an invalid opening_hours value, it will always be open",
```

Comment:
Can you generate a link to the OSM entity? There is `OsmWithTags#getOpenStreetmapLink()`. (This method should really be renamed to `getOpenStreetmapUrl()`)